_data/sidebars/home_sidebar.yml

```diff
@@ -154,6 +154,30 @@ entries:
           url: /participants.html
           output: web
 
+      - title: Features of Collections and Admin Sets
```

Comment:
I'd sort of like to see a page for collection type as well. Does this page exist anywhere?